8.5-java11/Dockerfile

```diff
@@ -31,6 +31,17 @@ COPY tmp/shared/app_insights/ApplicationInsights.xml /usr/local/app_insights/tom
 
 RUN apk add --update openssh-server bash openrc \
         && rm -rf /var/cache/apk/* \
+        # Remove unnecessary services
+        && rm -f /etc/init.d/hwdrivers \
```

Comment:
>hwdrivers [](start = 29, length = 9)

do we know what this one is used for ?

Reply:
This is one of the services from openrc. I don't exactly know what this is used for, but the only reason we need openrc is to launch the ssh daemon. ssh works fine without these services. 

---
In reply to: [264959397](https://github.com/Azure-App-Service/tomcat/pull/29#discussion_r264959397) [](ancestors = 264959397)